subschemas/substrate-chain/src/resolvers/Query/council.ts

```diff
@@ -28,35 +28,47 @@ export async function council(
     api.derive.chain.bestNumber(),
   ]);
 
+  const {address: addressFilter} = args;
```

Comment:
just one tine nitpicking suggestion:
I would do this on the params of the function:
`{addressFilter}: {address?: string | null}`
And delete this line.

But it’s about personal taste :slightly_smiling_face: